examples/get_started/03_figure_element.py

```diff
@@ -0,0 +1,92 @@
+"""
+3. Figure elements
+===============
+The figure shows the naming of figure elements in PyGMT.
+
+- :meth:`pygmt.Figure`: having a number of plotting methods. Every plot or map
+  must start with the creation of a :meth:`pygmt.Figure` instance
```

Comment:
Users can add different plots or / and maps to one figure object.
```suggestion
- :meth:`pygmt.Figure`: having a number of plotting methods. Every new figure
  must start with the creation of a :meth:`pygmt.Figure()` instance
```